### ISSUE TITLE
the results and training loss 

### ISSUE THREAD

**Issue by DingGitTemp** [sp-uhh/sgmse#42]
I attempted to train and test the model on the voicebank-demand dataset, but the results were not satisfactory. The enhanced speech couldn't be recognized as human voice . Are there any parameters that need to be reset?Additionally, during the training process, the loss of the training set consistently remained around 700. Is this normal?

**Comment by cobalamin** [comment]
Hi, did you downsample your version of the VB-DMD dataset to 16 kHz? The model is by default designed for 16 kHz.

**Comment by DingGitTemp** [comment]
The issue has been resolved; it turns out I hadn't downsampled the data to 16K. Thank you very much for your response; this truly is a remarkable piece of work.

**Comment by cobalamin** [comment]
Thanks! Happy to hear that it works now.
